modules/user_passwords/utils/scrypt.ts

```diff
@@ -0,0 +1,16 @@
+import { hash as hashSCrypt, verify } from "https://deno.land/x/scrypt@v4.4.4/mod.ts";
+import { ScryptParameters } from "https://deno.land/x/scrypt@v4.4.4/lib/format.ts";
+
+const scryptParams: Partial<ScryptParameters> = {
+    logN: 15,
```

Comment:
OWASP recommended is 14, any reason for 15?

Reply:
Respectfully, we were **both** way off!
Most recent recommendation is 

https://cheatsheetseries.owasp.org/cheatsheets/Password_Storage_Cheat_Sheet.html#scrypt

I'll go with `logN=15, r=8, p=3` because of possible memory limits on workers.